src/hl7v2/schema.clj

```diff
@@ -49,6 +50,27 @@
          (drop (if (node-attrs node) 2 1))
          (into []))))
 
+(defn node-annotation [node]
+  (letfn [(rm-diacritics [s]
+            (when s
+              (-> s
+                  (Normalizer/normalize Normalizer$Form/NFD)
+                  (str/replace #"[\u0300-\u036f]" ""))))]
+    (->> (node-children node)
+         (some #(when (and (= "documentation" (first %))
+                           (= "en" (:lang (second %))))
+                  (->> (str/split (last %) #"\s+")
+                       (map (fn [s]
+                              (-> (rm-diacritics s)
+                                  (str/replace #"/" "Or")
+                                  (str/replace #"-" ""))))
+                       (str/join "")))))))
+
+(defn type-base [node]
+  (->> (node-children node)
+       (filter (comp #{"extension"} node-tag))
+       (some (comp :base node-attrs))))
+
 (defn content-node? [node]
   (when-let [attrs (node-attrs node)]
```

Comment:
when-let qué hace? Busqué en clojuredocs, pero no comprendí muy bien.

Reply:
`(when-let` macro, combines `(let` and `(when`

```
(let [foo (:foo m)]
    (when foo
        (println foo)))
```

may be rewritten to 

```
(when-let [foo (:foo m)]
    (println foo))
```